### ISSUE TITLE
eslint-plugin-better-tailwindcss does not support Tailwind's opacity modifier syntax with slash (/)

### ISSUE THREAD

**Issue by menthol** [schoero/eslint-plugin-better-tailwindcss#114]
The eslint-plugin-better-tailwindcss incorrectly flags Tailwind CSS classes that use the opacity modifier syntax with slash (/). This syntax is valid and officially documented in Tailwind CSS.

The plugin generates an "Unregistered class detected" error for classes like `shadow-lg/50`, where `/50` specifies a 50% opacity level for the shadow effect.

This syntax is perfectly valid in Tailwind CSS and is used to modify the opacity of various properties such as shadows, text colors, background colors, etc.

The issue has been detected specifically with shadow utilities (like `shadow-lg/50`), but may affect other utility classes that use the slash syntax for opacity modifiers as well.
## Example
In the following code:
``` vue
<div class="shadow-lg/50 rounded-lg p-4 bg-white dark:border dark:border-black dark:bg-neutral-800">
  <!-- content -->
</div>
```
The class `shadow-lg/50` should be valid as it uses Tailwind's official syntax to apply a large shadow (`shadow-lg`) with 50% opacity.
## Error Message
``` 
ESLint: Unregistered class detected: shadow-lg/50 (better-tailwindcss/no-unregistered-classes)
at: shadow-lg/50
```
## Environment
- eslint-plugin-better-tailwindcss: 3.1.0
- Tailwind CSS: latest version
- Vue 3
- Nuxt 3.16.0

## Expected Behavior
The plugin should correctly recognize and validate Tailwind classes that use the opacity modifier syntax with slash, such as `shadow-lg/50`, `bg-blue-500/75`, etc.
